src/App/App.js

```diff
@@ -54,6 +55,13 @@ function App({ dexieCurrentUserInstance }) {
     toast.error(...getToastArguments(language.error.pushSyncErrorMessageStatusCode500))
   }
 
+  const handleUserDeniedSyncPull = useCallback(
+    (projectName) => {
+      history.push(projectName ? `/noProjectAccess/${projectName}` : '/noProjectAccess')
+    },
+    [history],
+  )
+
```

Comment:
We're passing `handleUserDeniedSyncPull` to where it gets called in a similar fashion to how we do so for `handleUserDeniedSyncPush`